tests/test_sync.py

```diff
@@ -0,0 +1,221 @@
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import os
+
+from playwright import Error
+from playwright.sync import (
+    SyncBrowser,
+    SyncConsoleMessage,
+    SyncDownload,
+    SyncWorker,
+    browser_types,
+    SyncPage,
+)
+
+
+def test_sync_query_selector(browser_name, launch_arguments):
+    browser = browser_types[browser_name].launch(**launch_arguments)
+    context = browser.newContext()
+    page = context.newPage()
+    page.setContent(
+        """
+    <h1 id="foo">Bar</h1>
+    """
+    )
+    assert (
+        page.querySelector("#foo").innerText() == page.querySelector("h1").innerText()
+    )
+    browser.close()
```

Comment:
```py
@pytest.fixture
def browser(browser_name, launch_arguments):
    browser = browser_types[browser_name].launch(**launch_arguments)
    yield browser
    browser.close()
```
seems nicer than having to create it and close it with each test

Reply:
definitely!